src/plugins/status_page/public/status_page.js

```diff
@@ -36,6 +36,14 @@ const chrome = require('ui/chrome')
         ui.serverState = overall.state;
         ui.serverStateMessage = overall.title;
       }
+
+      // Unless the status page was explicitly requested, there is no
+      // reason to stay on the status page if the status is green; reload
+      // the window so the user is shown the page they were requesting.
+      const statusPageUrl = chrome.addBasePath('/status');
+      if ((overall.state === 'green') && ($window.location.pathname !== statusPageUrl)) {
+        return $window.location.reload();
+      }
```

Comment:
Filed issue for the right fix at https://github.com/elastic/kibana/issues/7432 and removed reloading code block in db780bd645d4b120889facf42dc6b8a766a31460.